governance/votes/Executive vote - April 18, 2024.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Template - [Executive Vote] March 2024 AD Compensation, Q1 2024 AVC Member Participation Reward, Aave Revenue Share Transfer, Whitelist New Address in the RWA015-A Output Conduit, Push USDP out of Input Conduit, Trigger Spark Proxy Spell - April 18, 2024
+summary:  March 2024 AD Compensation, Q1 2024 AVC Member Participation Reward, Spark Protocol-Aave Share Transfer, Whitelisting new address in the RWA015-a output conduit, pushing 84,211.27 USDP out of the MCD PSM PAX input conduit jar, triggering a Spark proxy spell on Ethereum and Gnosis.
+date: 2024-04-18T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] [Executive Vote] March 2024 AD Compensation, Q1 2024 AVC Member Participation Reward, Aave Revenue Share Transfer, Whitelist New Address in the RWA015-A Output Conduit, Push USDP out of Input Conduit, Trigger Spark Proxy Spell - April 18, 2024
+
+The Governance Facilitators, SideStream, Dewiz, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- A total of **219.36 MKR** will be distributed to eight Aligned Delegates as compensation for March 2024.
```

Comment:
:red_circle: **blocking**

This value is incorrect and does not match the total in line 42.

```suggestion
- A total of **214.55 MKR** will be distributed to eight Aligned Delegates as compensation for March 2024.
```